Instagram/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,68 @@
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+
+@interface LoginViewController ()
+
+@property (strong, nonatomic) IBOutlet UITextField *usernameField;
+@property (strong, nonatomic) IBOutlet UITextField *passwordField;
+- (IBAction)didTapLogin:(UIButton *)sender;
+- (IBAction)didTapSignUp:(UIButton *)sender;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (IBAction)didTapSignUp:(UIButton *)sender {
+    PFUser *newUser = [PFUser user];
+    
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
+    
+    if ([self.usernameField.text isEqual:@""] || [self.passwordField.text isEqual:@""]) {
+        // TODO: alert
+    } else {
+        [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * error) {
+            if (error != nil) {
+                NSLog(@"Error: %@", error.localizedDescription);
```

Comment:
Should avoid checking in log statements since it is easy to forget to remove them.